mlflow/keras/load.py

```diff
@@ -45,7 +45,8 @@ def predict(self, data, **kwargs):
                 f"received type: {type(data)}.",
                 INVALID_PARAMETER_VALUE,
             )
-        return model_call(data)
+        # Return value must be json serializable, so we convert the Tensor output to a list.
+        return keras.ops.convert_to_numpy(model_call(data)).tolist()
```

Comment:
Should we fix this in _get_jsonable_obj instead of here? It's only for serving right?

Reply:
ohh could you share a bit more about `_get_jsonable_obj `? I haven't seen it before. 

This wrapper is only for pyfunc loading, so it makes sense for me to have it return a list than a tensor.

Reply:
`_get_jsonable_obj` is called inside `predictions_to_json` where we wrap the pyfunc predict result inside serving.